package.json

```diff
@@ -112,11 +94,9 @@
   "dllPlugin": {
     "name": "vuePluginTemplateDeps",
     "include": [
-      "mocha/mocha.js",
       "style-loader!css-loader!mocha-css",
       "html-entities",
       "vue/dist/vue.js",
-      "chai",
```

Comment:
I'm admittedly not sure what this Webpack plugin configuration does without looking into it. I feel like we might be able to simplify some of the Webpack build process, but I think this should be ok for now?